src/components/campaign/ActionForm.jsx

```diff
@@ -9,6 +11,26 @@ export default class ActionForm extends React.Component {
         response: React.PropTypes.bool,
     };
 
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            viewMode: undefined,
+        };
+    }
+
+    componentDidMount() {
+        let node = ReactDOM.findDOMNode(this.refs.infoText);
+        if (node) {
+            console.log(node.clientHeight);
```

Comment:
Try to not commit code that generates console output. I keep forgetting these in there myself, but I try to make a habit of searching project-wide for "console" before each commit.